app/src/androidTest/java/org/mozilla/fenix/ui/robots/HistoryRobot.kt

```diff
@@ -73,13 +75,22 @@ class HistoryRobot {
 
     fun clickDeleteAllHistoryButton() = deleteButton().click()
 
+    fun selectEverythingOption() = deleteAllHistoryPromptEverythingOption().click()
+
     fun confirmDeleteAllHistory() {
         onView(withText("Delete"))
             .inRoot(isDialog())
             .check(matches(isDisplayed()))
             .click()
     }
 
+    fun cancelDeleteAllHistory() =
```

Comment:
Nit: rename this for more general usage as "cancelDeleteHistory" (in case we want to cancel deleting history from the last hour for example)

Reply:
Done !